### PR TITLE
Mat 4133 UI portion of verifying impact on test cases when population definitions change in measure group

### PR DESCRIPTION
**Pull request by bwelch8105**

### CODE REVIEW

File: cypress/Shared/TestCasesPage.ts [hunk 1]
```diff
@@ -2,10 +2,10 @@ import {EditMeasurePage} from "./EditMeasurePage"
 
 export class TestCasesPage {
 
-    public static readonly newTestCaseButton = '[data-testid="create-new-test-case-button"] > .sc-iqseJM'
+    public static readonly newTestCaseButton = '[data-testid="create-new-test-case-button"]'// > .sc-iqseJM'
     public static readonly testCaseDescriptionTextBox = '[data-testid=create-test-case-description]'
     public static readonly testCaseSeriesTextBox = '[data-testid="create-test-case-series"] > .MuiOutlinedInput-root'
-    public static readonly existingTestCaseSeriesDropdown = '#mui-4-option-0'
+    public static readonly existingTestCaseSeriesDropdown = '#mui-6-option-0'//'#mui-4-option-0'
```
Comment (cvasile001) [inline cypress/Shared/TestCasesPage.ts:8]: Can you work with the developer and get an ID added for this element?

File: cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts [hunk 2]
```diff
@@ -71,8 +71,12 @@ describe('Test Case Expected Measure Group population values based on initial me
         cy.get(TestCasesPage.testCasePopulationValuesTable).should('be.visible')
         cy.get(TestCasesPage.testCasePopulationValues).should('contain.text', 'PopulationExpectedActual')
 
-        //validate proper Population Value check boxes, based on value of score
-        Utilities.validateTCPopValueCheckBoxes((measureScoringArray[3].valueOf()).toString())
+        cy.get(TestCasesPage.testCaseIPPCheckBox).should('be.visible')
+        cy.get(TestCasesPage.testCaseIPPCheckBox).check().should('be.checked')
```
Comment (cvasile001) [inline cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts:75]: What does the .check() do? Just curious

File: cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts [hunk 3]
```diff
@@ -202,4 +206,132 @@ describe('Test Case Expected Measure Group population values based on initial me
         cy.get(TestCasesPage.testCaseIPPCheckBox).should('not.be.checked')
 
     })
+    it('Test Case Population value options are limited to those that are only defined from Measure Group -- only required Populations on Measure Group', () => {
```
Comment (cvasile001) [inline cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts:209]: is it possible to make this title more concise?

**Review by cvasile001 — CHANGES_REQUESTED** [review]

File: cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts (hunk 2 above)
Comment (bwelch8105) [reply cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts:75]: The '.check()' checks the check box. In these two lines, I am 1.) making sure the checkbox for the Initial Population check box is visible; 2.) checking the check box; 3.) verifying that after checking the check box that it is, in fact, checked

File: cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts (hunk 3 above)
Comment (bwelch8105) [reply cypress/integration/WebInterface/Test Cases/TestCasePopulationValues.spec.ts]: I have made the newer tests a bit more concise with their names / titles.

**Pull request merged**